packages/react-instantsearch-hooks/src/__tests__/InstantSearchSSRContext.test.tsx

```diff
@@ -0,0 +1,7 @@
+import { InstantSearchSSRContext } from '../InstantSearchSSRContext';
+
+describe('InstantSearchSSRContext', () => {
+  test('exposes a displayName', () => {
```

Comment:
The `displayName` should be gone in production, you could test it by temporarily setting `__DEV__` to a falsy value.

Reply:
We cannot override Jest `globals` unfortunately.